otx/algorithms/action/adapters/mmaction/models/backbones/movinet.py

```diff
@@ -0,0 +1,1102 @@
+"""
+Code inspired by:
+https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv2.html
+https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv3.html
```

Comment:
It seems like you copied the model code from the open source. I think you need to add the original reference url.

Reply:
Reflected, thank you.